public/app/plugins/datasource/loki/queryUtils.ts

```diff
@@ -304,6 +305,30 @@ export function getStreamSelectorsFromQuery(query: string): string[] {
   return labelMatchers;
 }
 
+export function removeTrailingPipelines(query: string): string {
+  const trailingPipelinePositions: Position[] = [];
```

Comment:
If I understand this correctly, this is not just trailing pipelines but all pipelines in the query, right?